src/app/actions/index.ts

```diff
@@ -23,3 +24,16 @@ const wrappedGetTodayPrices = ({
   })
 );
 export { wrappedGetTodayPrices as getTodayPrices };
+
+const wrapperCreateLinkToken = ({
+  userId,
+}: {
+  userId: string,
+}): ReturnType<typeof createLinkToken> => (
+  createLinkToken({
+    accessToken: Actions.accessToken,
+    userId,
+  })
+);
```

Comment:
the wrapped functions can probably be generated dynamically, once we start having more, definitely something to look at